pkg/kubelet/kubelet_getters.go

```diff
@@ -258,3 +258,19 @@ func (kl *Kubelet) GetCachedMachineInfo() (*cadvisorapiv1.MachineInfo, error) {
 	}
 	return kl.machineInfo, nil
 }
+
+// ReflectorsHealthy returns a boolean that is true when all of the kubelet's
+// reflectors are healthy. If the boolean is false then the reflector errors
+// are returned as well.
+func (kl *Kubelet) ReflectorsHealthy() (bool, []error) {
+	reflectorErrors := make([]error, 0, len(kl.reflectors))
```

Comment:
nit: since the common case should be 0 errors, skip the allocation. i.e. `var reflectorErrors []eror`